packages/reaction-product-variant/client/templates/products/productDetail/variants/variantList/variantList.js

```diff
@@ -6,31 +6,38 @@ Template.variantList.helpers({
     let inventoryTotal = 0;
     const variants = ReactionProduct.getTopVariants();
 
-    if (variants.length > 0) {
+    if (variants.length) {
       // calculate inventory total for all variants
-      for (let variant of variants) {
-        let qty = ReactionProduct.getVariantQuantity(variant);
+      variants.forEach(variant => {
```

Comment:
Generally I am strongly against changes to working code that do nothing but change the method by something is accomplished if there is no driving reason beyond the author's preferred style. (If I am missing a reason here, let me know).

I addition, I would say that we should/do prefer the easier to read iterator version over the `forEach` version.